internal/pkg/service/buffer/model/model.go

```diff
@@ -0,0 +1,215 @@
+package model
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/c2h5oh/datasize"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+type ImportCondition struct {
+	Count int               `json:"count" validate:"min=1,max=10000000"`
+	Size  datasize.ByteSize `json:"size" validate:"min=100,max=50000000"`
+	Time  time.Duration     `json:"time" validate:"min=30s,max=24h"`
```

Comment:
offtopic: na tie validacie budeme musiet dopisat unit testy, `time.Duration` je interne cislo, takato validacia nebude fungovat.

Reply:
Chtělo by to napsat custom validator, udělám to v nějakým dalším PR

Reply:
for ref: https://github.com/go-playground/validator/issues/584

Reply:
Přijde mi, že to dává smysl, protože je to dost extra nul, u kterých může být jednoduše překlep 🙂

Reply:
jj